src/main/webapp/js/map-loader.js

```diff
@@ -45,6 +45,34 @@ function createMap(){
   fetchMarkers();
 }
 
+/**
+* Amsterdam main function
+*/
+function createAmsterdamMap(){
+	map = new google.maps.Map(document.getElementById('map'), {
+		center: {lat: 52.3680, lng: 4.9036},
+		zoom: 12
+	});
+	map.addListener('click', (event) => {
+	createMarkerForEdit(event.latLng.lat(), event.latLng.lng());
+	})
+	fetchMarkers();
+}
+
+/**
+* Venice main function
+*/
+function createVeniceMap(){
+	map = new google.maps.Map(document.getElementById('map'), {
+	center: {lat: 45.4408, lng: 12.3155},
```

Comment:
The code is repeated so maybe create one common function and pass the lat and lng as the params to remove redundant code.

Reply:
yes good idea, fixed